rundb

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Why can't we run `foreman start` instead of customising the run script?

Reply:
My god, we can run this.

Reply:
The `rundb` script does some other things around `foreman start` which Docker does not do for us.  It needs to make sure the MySQL daemon is up, that `rails` listens on the port, etc.